python/paddle/v2/fluid/layers/control_flow.py

```diff
@@ -270,6 +270,7 @@ def get_parameters(self):
                 for in_var_name in op.input(iname):
                     if in_var_name not in local_inputs:
                         params.append(in_var_name)
+        params = list(set(params))
```

Comment:
@reyoung Duplications of the parameter names will be taken care by backward. parallel_do only needs a unique list.